Apps/W1/EssentialBusinessHeadlines/app/src/codeunits/EssBusHeadlineSubscribers.Codeunit.al

```diff
@@ -37,7 +37,8 @@ codeunit 1438 "Ess. Bus. Headline Subscribers"
             exit;
 
         EssentialBusinessHeadline.SetRange("User Id", UserSecurityId());
-        EssentialBusinessHeadline.DeleteAll();
+        if not EssentialBusinessHeadline.IsEmpty then
```

Comment:
Thanks, I was unsure since IsEmpty is not necessary with parentheses. I will do in further PR's with them in place

Reply:
@JesperSchulz can it also be a support case to AL team to add then () to isEmpty when we do development if you think we should add it?
![image](https://github.com/microsoft/ALAppExtensions/assets/106300783/5fc0fc61-456e-49b2-a811-36ab7c8b6863)

but when we add it to the code:

![image](https://github.com/microsoft/ALAppExtensions/assets/106300783/b71a4aaf-6422-4029-934a-8c52177a1434)

So there is inconsistency - this is also why we never add () to IsEmpty since AL is not adding it.